components/Wave/Wave.jsx

```diff
@@ -0,0 +1,52 @@
+import styles from './wave.module.scss'
+
+import React from 'react'
+
+export const WaveHeader = () => {
+  return (
+    <div className={styles.wave}>
+        <div className={styles.wave__desktop}>
```

Comment:
Why do you need to declare the styles more than once?... can you just just have wave__desktop? not styles.wave__desktop ?